resources/lang/en-US/admin/settings/general.php

```diff
@@ -401,6 +401,9 @@
     'due_checkin_days_help' => 'How many days before the expected checkin of an asset should it be listed in the "Due for checkin" page?',
     'no_groups' => 'No groups have been created yet. Visit <code>Admin Settings > Permission Groups</code> to add one.',
     'text' => 'Text',
+    'manager_view' => 'Manager View',
+    'manager_view_enabled_text' => 'Enable Manager View',
+    'manager_view_enabled_help' => 'Allow managers to view assets assigned to their direct and indirect reports in their account view.',
```

Comment:
Since it's not just assets but all assigned items maybe this should be
```suggestion
    'manager_view_enabled_help' => 'Allow managers to view assigned items to their direct and indirect reports in their account view.',
```